src/docs/asciidoc/reference/manage/application-add.adoc

```diff
@@ -0,0 +1,29 @@
+:relative-path: ../../../
+include::{docdir}/variables.adoc[]
+
+There are several ways to create a new application :
+
+* link:https://www.zetapush.com/sign-up-for-a-free-trial/[Signup for a free trial]
+* link:https://www.zetapush.com/contact/[Contact us for more suitable application] 
+* Via the *web console* (no available yet)
+* Via the *CLI* (no available yet)
+
+== Sign up for a free trial
+
+When you ask for a free trial, a *ZetaPush account* and an *application* are provided to you.
+
+You can ask for a free trial via link:https://www.zetapush.com/sign-up-for-a-free-trial/[our web site].
+
+NOTE: Now, when you asking for a free trial, the *ZetaPush account* and the *application* are manually created. Soon we will open access to everyone for get a free trial of 30 days. In this case the account and the application will be automatically created.
+
+== Contact us for more suitable application
+
+If you want to have a more suitable *ZetaPush account* (and *application*), you can link:https://www.zetapush.com/contact/[contact us] to ensure a clear understanding of your needs.
+
+== Create application via web console
+
+NOTE: The creation of an application via the *web console* is not available yet.
+
+== Create application via CLI
+
+NOTE: The creation of an application via the *CLI* is not available yet.
```

Comment:
Note convention

Maybe we should explain that we have deactivated this feature in order to assist our clients